src/Dijkstra/AST/RWS.agda

```diff
@@ -154,6 +154,8 @@ ASTPredTrans.opPT RWSPT (RWSlisten{A'} refl) f P (ev , st) =
 ASTPredTrans.opPT RWSPT{A} RWSpass f P (ev , st) =
   f (Level.lift unit) (RWSpassPost P) (ev , st)
 
+ptsRWS = ASTPredTrans.predTrans RWSPT
```

Comment:
I'm confused - didn't Christa already merge this (and `sufRWS`) into `main-ast`?

Reply:
I was also confused by this, and was too tired at the time.  I think it will not be replicated when I create the requested separate PRs.